Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H

```diff
@@ -134,6 +134,25 @@ struct CylindricalYeeAlgorithm {
         return inv_dr*( F(i,j,k,comp) - F(i-1,j,k,comp) );
     }
 
+    /** Applies the differential operator `1/r * d(r * dF/dr)/dr`,
+     * where `F` is on a *cell-centered* or a nodal grid in `r`
+     * The input parameter `r` is given at the cell-centered position */
+    template< typename T_Field>
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    static amrex::Real Dr_rDr_over_r (
+        T_Field const& F,
+        amrex::Real const r, amrex::Real const dr,
+        amrex::Real const * const coefs_r, int const n_coefs_r,
```

Comment:
For consistency with `Dzz`:
```suggestion
        amrex::Real const * const coefs_r, int const /*n_coefs_r*/,
```